src/components/slider/index.tsx

```diff
@@ -621,6 +622,9 @@ const styles = StyleSheet.create({
     shadowRadius: SHADOW_RADIUS,
     elevation: 2
   },
+  disabledThumb: {
+    borderColor: Colors.$backgroundElevated
```

Comment:
Why this token is only for the disabled thumb? The active one is white (as defined in line 618), shouldn't it be tokenized as well?

Reply:
For the active slider, the thumb should be white for both dark and light modes. 
For the disabled slider, it should be elevated (white in light mode and grey1 in dark)